### ISSUE TITLE
media_api.go auto-generated file error

### ISSUE THREAD

**Issue by grokify** [grokify/go-voicebase-v3#2]
The auto-generated `media_api.go` file has some errors:

```bash
../../media_api.go:886: undefined: file
../../media_api.go:888: undefined: file in file.Name
```

* https://github.com/grokify/go-voicebase-v3/blob/6a2b2e67696b33cdacc9c4cbf39a4f93339f2d28/media_api.go#L886
* https://github.com/grokify/go-voicebase-v3/blob/6a2b2e67696b33cdacc9c4cbf39a4f93339f2d28/media_api.go#L888

```go
fbs, _ := ioutil.ReadAll(file)
localVarFileBytes = fbs
localVarFileName = file.Name()
```

**Comment by grokify** [comment]
This has been fixed in 0ecc5bedab8cb4354697fb21145178807ca12761

**Comment by grokify** [comment]
This Swagger is built using the following Swagger Spec:

https://github.com/grokify/go-voicebase-v3/blob/8e410c1c07aa89d4fbe0e4f2cb486f83f706f3ca/swagger_voicebase_v3.yaml
